build-w-docker.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+# Exit immediately if a command exits with a non-zero status.
+set -e
+
+# Lines added to get the script running in the script path shell context
+# reference: http://www.ostricher.com/2014/10/the-right-way-to-get-the-directory-of-a-bash-script/
+cd $(dirname $0)
+
+# To avoid issues with MINGW y Git Bash, see:
+# https://github.com/docker/toolbox/issues/673
+# https://gist.github.com/borekb/cb1536a3685ca6fc0ad9a028e6a959e3
+export MSYS_NO_PATHCONV=1
+export MSYS2_ARG_CONV_EXCL="*"
+
+rm -rf dev
```

Comment:
It is not necessary because this folder is not mapped inside container (my fault sorry)

Reply:
Only dev folder is not mapped inside container? Or dev and dist folder? Thanks.

Reply:
Only _dist_ folder is mapped:

![image](https://user-images.githubusercontent.com/1157864/52874126-705edd80-312f-11e9-89b8-5a8f9a937084.png)

Reply:
You're right, thanks Andres!